english/fork-knots/melt-with-fork.md

```diff
@@ -1,2 +1,14 @@
 You start tying knots on the fork. Then stick the fork into a marshmallow wall
-and light a free end of a rope up. Yes, the wall slowly melts around the hot fork!
\ No newline at end of file
+and light a free end of a rope up. Yes, the wall slowly melts around the hot fork!
+
+As you are about to step outside you notice an altar in the middle of the room with a chalice on it.
+
+Do you?
+
+[Quickly run outside and explore?](..explore-outside/explore-outside.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.